earthpy/tests/test_hillshade.py

```diff
@@ -0,0 +1,61 @@
+""" Tests for the hillshade function. """
+
+import numpy as np
+import pytest
+import earthpy.spatial as es
+
+
+@pytest.fixture
+def hillshade_arr():
+    """Create three-dimensional array for testing input array
+    to hillshade function. """
+    arr = np.array(
+        [
+            [
+                [1100, 1500, 1200, 1000, 800, 700],
+                [1000, 1400, 1300, 1200, 1000, 900],
+            ]
+        ]
+    )
+    return arr
+
+
+def test_hillshade_shape(hillshade_arr):
+    """If provided with input array that is two-dimensional,
+    ValueError returned."""
+
```

Comment:
If you make a 2-dim arr for the fixture then add here

`hillshade_arr[np.newaxis, :]`
to add a third dimension

Docstring:
`A 2-dimensional arr should return a ValueError`